DOCS/new-license-workflow.md

```diff
@@ -4,42 +4,41 @@
 
 New license requests are to be assigned to an SPDX legal team member and that person is responsible for following the request through to its final determination.  Make sure to add yourself as the Assignee in the license request issue here on GitHub, add the [label](https://github.com/spdx/license-list-XML/labels) `new license/exception request`, and add the appropriate release milestone to the issue.
 
 ## Things to check for initial request:
 
 1. Is the license or exception already on the SPDX License List?  Has it been submitted and rejected previously? (If yes, go to "If license not accepted")
-2. Is the license or exception similar enough to an existing license or exception that additional markup could accomodate a match and there is no need to add the license?
+2. Is the license or exception similar enough to an existing license or exception that additional markup could accommodate a match and there is no need to add the license?
     1. Use Alan’s diff tool to compare to existing licenses.
     2. If it’s a case that additional markup would create match, then may want to discuss with legal team to ensure markup is non-substantive or differences in text do not alter legal meaning (if so, this cuts towards adding a new license). For more on this see the [Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines), guideline #2 in particular.
-    3. If additional markup can accommodate the license, then the license does not need to be added: inform the requestor, comment on the issue as such, then create a PR for the existing license with the additional markup, and close issue once the PR has been merged.
-
-2. If the submitter is not the license author or steward, ask for that contact or try to find that person or organization to make them aware the license has been submitted.
-3. Check the submission for any other missing information, e.g., working URL, examples of use, full text, standard header, etc. See the Field definitions on [Overview page](https://spdx.dev/license-list/) for assistance.
-Ask the submitter for any additional info needed, preferably via the Github issue, if possible. Record any updates there.
+    3. If additional markup can accommodate the license, then the license does not need to be added: inform the requester, comment on the issue as such, then create a PR for the existing license with the additional markup, and close issue once the PR has been merged.
+3. If the submitter is not the license author or steward, ask for that contact or try to find that person or organization to make them aware the license has been submitted.
+4. Check the submission for any other missing information, e.g., working URL, examples of use, full text, standard header, etc. See the Field definitions on [Overview page](https://spdx.dev/license-list/) for assistance.
+Ask the submitter for any additional info needed, preferably via the GitHub issue, if possible. Record any updates there.
```

Comment:
just noticed, that is an outdated link, it should be "Explanation of SPDX License List Fields" with the link https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md

Reply:
There we go, @jlovejoy! :) Added commit 08a097cdf6e5b854c574d7b3022b0d9a01f69458.